docs/registry/attributes/peer.md

```diff
@@ -9,4 +9,17 @@ Operations that access some remote service.
 
 | Attribute | Type | Description | Examples | Stability |
 |---|---|---|---|---|
-| <a id="peer-service" href="#peer-service">`peer.service`</a> | string | The [`service.name`](/docs/resource/README.md#service) of the remote service. SHOULD be equal to the actual `service.name` resource attribute of the remote service if any. | `AuthTokenCache` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="peer-service" href="#peer-service">`peer.service`</a> | string | The [`service.name`](/docs/resource/README.md#service) of the remote service. SHOULD be equal to the actual `service.name` resource attribute of the remote service if any. [1] | `AuthTokenCache` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+**[1] `peer.service`:** It MUST be possible to configure the peer service name in declarative configuration as follows:
+
+```yaml
+instrumentation/development:
```

Comment:
why would it be an instrumentation responsibility at all BTW? There are a lot of HTTP / RPC/ etc instrumentations producing spans that may benefit from `peer.service`.

This enrichment can be done in the single processor (collector or per-language) and dozens of instrumentations should never bother.